cmd/main.go

```diff
@@ -144,6 +144,10 @@ func main() {
 		numaLogger.Fatal(err, "Failed to start configmap watcher")
 	}
 
+	if err := kubernetes.SetDynamicClient(newRawConfig); err != nil {
+		numaLogger.Fatal(err, "Failed to set dynamic client set")
+	}
```

Comment:
Creating the raw client initially, instead of generating the clientset all the time

Reply:
I have reverted this change, as it was causing test case failure somehow, I'll raise followup PR for it.